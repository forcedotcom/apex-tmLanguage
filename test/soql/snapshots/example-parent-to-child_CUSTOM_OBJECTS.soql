SELECT Name, (SELECT Name FROM Line_Items__r) FROM Merchandise__c WHERE Name LIKE ‘Acme%’
