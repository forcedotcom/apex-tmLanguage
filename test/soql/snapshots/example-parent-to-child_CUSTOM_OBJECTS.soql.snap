>SELECT Name, (SELECT Name FROM Line_Items__r) FROM Merchandise__c WHERE Name LIKE ‘Acme%’
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^ source.soql keyword.query.field.apex
#           ^ source.soql punctuation.separator.comma.apex
#            ^ source.soql
#             ^ source.soql punctuation.parenthesis.open.apex
#              ^^^^^^ source.soql keyword.operator.query.select.apex
#                    ^ source.soql
#                     ^^^^ source.soql keyword.query.field.apex
#                         ^ source.soql
#                          ^^^^ source.soql keyword.operator.query.from.apex
#                              ^ source.soql
#                               ^^^^^^^^^^^^^ source.soql storage.type.apex
#                                            ^ source.soql punctuation.parenthesis.close.apex
#                                             ^ source.soql
#                                              ^^^^ source.soql keyword.operator.query.from.apex
#                                                  ^ source.soql
#                                                   ^^^^^^^^^^^^^^ source.soql storage.type.apex
#                                                                 ^ source.soql
#                                                                  ^^^^^ source.soql keyword.operator.query.where.apex
#                                                                       ^ source.soql
#                                                                        ^^^^ source.soql keyword.query.field.apex
#                                                                            ^ source.soql
#                                                                             ^^^^ source.soql keyword.operator.query.apex
#                                                                                 ^ source.soql
#                                                                                  ^ source.soql
#                                                                                   ^^^^ source.soql keyword.query.field.apex
#                                                                                       ^ source.soql keyword.operator.arithmetic.apex
#                                                                                        ^^ source.soql
>